src/base.css

```diff
@@ -1357,6 +1359,22 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Styles to support the mention of Unicode characters and character sequences *****/
+   /* For more information, see: https://www.w3.org/TR/international-specs#char_ref  */
+
+	.codepoint bdi {
+		line-height: 1em;
+		font-size: 125%;
+		padding-inline: 0.25em;
+	}
+
+	.uname {
+		font-variant: all-small-caps;
+		font-size: 100%;                    
```

Comment:
The font size appears to have grown also.

Currently, respec_2022.css has the following:
````
.codepoint bdi {
    line-height: 1em;
    font-size: 1.6em;
    padding-inline: 0.25em;
	}

.uname {
	font-size: 75%;
	letter-spacing:0.03em;
	color: brown;
	}
````